mlflow/models/evaluation/default_evaluator.py

```diff
@@ -0,0 +1,330 @@
+import mlflow
+from mlflow.models.evaluation.base import (
+    ModelEvaluator,
+    EvaluationMetrics,
+    EvaluationArtifact,
+    EvaluationResult,
+)
+from mlflow.entities.metric import Metric
+from mlflow.utils.file_utils import TempDir
+from mlflow.tracking.artifact_utils import get_artifact_uri
+from sklearn import metrics as sk_metrics
+import matplotlib.pyplot as pyplot
+import scikitplot
+import shap
+import shap.maskers
+import math
+import pandas as pd
+import time
+from functools import partial
+
+shap.initjs()
```

Comment:
Can we run this inside of `DefaultEvaluator.evaluate()`?

Reply:
One other weird thing to note for this: 
When this is executed, Jupyter will generate an outside-of-cell visualization notifying the user that the cell that called this function has instantiated the js plugin. It's a weird graphic that pops up and I've had to field questions from people who wondered if something broke in their notebooks. Perhaps we should add something in the docstring to let users know that the (js) symbol that appears is expected behavior.

Reply:
What about temporarily patch `IPython.core.display.display` method (replace it with empty function) to avoid it display the logo.

Reply:
seems `initjs` only read some bundle file:
```
def getjs():
    bundle_path = os.path.join(os.path.split(__file__)[0], "resources", "bundle.js")
    with io.open(bundle_path, encoding="utf-8") as f:
        bundle_data = f.read()
    return "<script charset='utf-8'>{bundle_data}</script>".format(bundle_data=bundle_data)


def initjs():
    assert have_ipython, "IPython must be installed to use initjs()! Run `pip install ipython` and then restart shap."
    logo_path = os.path.join(os.path.split(__file__)[0], "resources", "logoSmallGray.png")
    with open(logo_path, "rb") as f:
        logo_data = f.read()
    logo_data = base64.b64encode(logo_data).decode('utf-8')
    display(HTML(
        "<div align='center'><img src='data:image/png;base64,{logo_data}' /></div>".format(logo_data=logo_data) +
        getjs()
    ))
```
does it do any other initial work ?

Reply:
Looks like it just loads that js lib to create the interactive plot. You could probably just load that bundle if it's needed to generate the image html for logging purposes and omit all of the other validations and logo display execution.